### ISSUE TITLE
TypeError: object.hasOwnProperty is not a function

### ISSUE THREAD

**Issue by Zaseth** [kriszyp/msgpackr#19]
Objects that are parsed from querystring can't be serialized unless they're cloned. One way to fix this is not to use `hasOwnProperty`.

Stack:
```
TypeError: object.hasOwnProperty is not a function
    at /node_modules/msgpackr/pack.js:427:33
    at pack (/node_modules/msgpackr/pack.js:366:7)
    at Packr.pack.encode (/node_modules/msgpackr/pack.js:102:5)
    at /index.js:6:1
    at Script.runInContext (vm.js:130:18)
    at Object.<anonymous> (/run_dir/interp.js:209:20)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)Hint: hit control+c anytime to enter REPL.
```

Code to reproduce:
```js
const { pack } = require('msgpackr')
const { parse } = require('querystring')

const stuff = parse('&items=[1]')

pack(stuff)
```

**Comment by kriszyp** [comment]
Fixed in v1.2.4

**Comment by Zaseth** [comment]
Thanks!
